Zend/zend_API.c

```diff
@@ -439,6 +439,24 @@ ZEND_API ZEND_COLD void zend_argument_value_error(uint32_t arg_num, const char *
 }
 /* }}} */
 
+ZEND_API ZEND_COLD void zend_argument_cannot_be_empty_error(uint32_t arg_num) /* {{{ */
```

Comment:
Not sure what I'm missing here, but why don't you just call zend_argument_value_error internally?

Reply:
I just didn't want to use the variadic version as I didn't have any args, but I could yes.

Reply:
I see. Since this is cold code anyway wi would prefer that over the current code, to avoid duplicating code and have a simpler implementation. This isn't performance sensitive anyway.

Reply:
Updated in-place in the first commit